internal/runner/outputter.go

```diff
@@ -87,6 +87,10 @@ func writePlainHostIP(_ string, results map[string]resolve.Result, writer io.Wri
 	sb := &strings.Builder{}
 
 	for _, result := range results {
+		if result.Host == "" {
```

Comment:
how about skipping it in https://github.com/projectdiscovery/urlfinder/blob/dev/internal/runner/enumerate.go#L53 ?

Reply:
yeah this is better